section11/numberformat_proposed.html

```diff
@@ -757,10 +790,11 @@ <h1>Internal slots</h1>
 
       <ul>
         <li>The list that is the value of the *"nu"* field of any locale field of [[LocaleData]] must not include the values *"native"*, *"traditio"*, or *"finance"*.</li>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a patterns field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of the four number format styles: *"decimal"*, *"percent"*, *"currency"*, and *"unit"*.</li>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a *"patterns"* field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of the four number format styles: *"decimal"*, *"percent"*, *"currency"*, and *"unit"*.</li>
```

Comment:
How should "patterns" be styled? @spectranaut @leobalter

Reply:
Maybe with `~` as internal values, https://tc39.es/ecma262/#sec-value-notation?

Reply:
In 402, it look we use [[field]] notation whenever referencing a field, see for example this paragraph: https://tc39.es/ecma402/#sec-bestfitmatcher

So far, we don't use `~` for internal values anywhere in the spec.

Reply:
SGTM.  Pushed a change to stylize this and related internal record keys as `[[field]]`